microcosm_flask/namespaces.py

```diff
@@ -32,11 +32,9 @@ class Namespace(object):
     object encapsulates the rest.
 
     """
-
     def __init__(self,
                  subject,
                  object_=None,
-                 path=None,
```

Comment:
Stop allowing "path" as an argument. It hasn't existed as a real argument for a long time.